e2e/LongHaul/device/IIotHub.cs

```diff
@@ -21,6 +21,6 @@ internal interface IIotHub
         /// </summary>
         /// <param name="properties">Twin properties to set.</param>
         /// <param name="cancellationToken">A cancellation token.</param>
-        Task SetPropertiesAsync(object properties, CancellationToken cancellationToken = default);
+        Task SetPropertiesAsync(SystemProperties properties, CancellationToken cancellationToken = default);
```

Comment:
This is used for `SystemProperties` only, so refactoring the parameter type.

Reply:
Rename the method?

Reply:
Actually, I think this is meant to be twin properties, so it should be ReportedProperties.

Reply:
Nope, I was wrong there too.

The idea was to set a twin property where the contents of the object were the key/value pairs. The v2 API doesn't allow that.